pages/api/tito-check-slug.ts

```diff
@@ -0,0 +1,48 @@
+export default async function (req, res) {
```

Comment:
The event slug should be passed to this function on the request query string. In the GET in `event-signup.js` The fetch should change the URI to be `/api/tito-check-slug?eventId=${event.titoSlug}`.

This will make it so you do not need to call the first fetch here to `/events`. All that method did was try to find 'the next' event which we now have from the request. L9-L21 can be removed.

Reply:
Resolved in commit: ee1641a8fbf0b7d95a0cc3cc58f9add5dac2ef7e